TOOLS/lua/autoload.lua

```diff
@@ -22,11 +25,30 @@ function Set (t)
     return set
 end
 
-EXTENSIONS = Set {
-    'mkv', 'avi', 'mp4', 'ogv', 'webm', 'rmvb', 'flv', 'wmv', 'mpeg', 'mpg', 'm4v', '3gp',
-    'mp3', 'wav', 'ogm', 'flac', 'm4a', 'wma', 'ogg', 'opus',
+function SetUnion (a,b)
+    local res = {}
+    for k in pairs(a) do res[k] = true end
+    for k in pairs(b) do res[k] = true end
+    return res
+end
+
+EXTENSIONS_VIDEO = Set {
+    'mkv', 'avi', 'mp4', 'ogv', 'webm', 'rmvb', 'flv', 'wmv', 'mpeg', 'mpg', 'm4v', '3gp'
+}
+
+EXTENSIONS_AUDIO = Set {
+    'mp3', 'wav', 'ogm', 'flac', 'm4a', 'wma', 'ogg', 'opus', 'jpg', 'png', 'tif', 'jpeg'
```

Comment:
Stray image formats

Reply:
damn, missed that, sorry

Reply:
Fixed in https://github.com/mpv-player/mpv/commit/102d4a609effeb0433474d21ebf0fe048d228a0b